### PR TITLE
Convert README to rst, use it in long_description

### PR DESCRIPTION
**Pull request by hroncok**
Fixes https://github.com/sclorg/rpm-list-builder/issues/90

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+    $ which rpmlb
+    /usr/local/python-3.6.1/bin/rpmlb
+
+    $ rpmlb -h
+
+Usage
+-----
+
+See `Users Guide <https://github.com/sclorg/rpm-list-builder/blob/master/docs/users_guide.md>`_.
```
Comment (hroncok) [inline README.rst]: Changed for full URL, so the link works on PyPI as well.

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+Run below command to check installed command.
+
+::
+
+    $ source venv/bin/activate
+    (venv) $ venv/bin/rpmlb -h
+    (venv) $ deactivate
+
+Install directly by pip
```
Comment (khardix) [inline README.rst]: Since we are/will be on PyPI, I would move this sub-section to the top of the install section and simplify it to `pip3 install rpmlb`.

Comment (junaruga) [inline README.rst]: I agree with your opinion.
But I think this PR's purpose is to convert from markdown to rst file to show it on pypi.
So, your thoughts can be another PR.

Comment (hroncok) [inline README.rst]: Agree with @junaruga. Also it's hard to track changes made during format conversion.

Comment (khardix) [inline README.rst]: OK, then I'm fine with this ☺.
